linalg/hypre.cpp

```diff
@@ -2228,10 +2228,10 @@ void HypreParMatrix::EliminateRows(const Array<int> &rows)
 
 void HypreParMatrix::EliminateBC(const HypreParMatrix &Ae,
                                  const Array<int> &ess_dof_list,
-                                 const Vector &X, Vector &B) const
+                                 const Vector &x, Vector &b) const
```

Comment:
I would prefer to keep capitals for the linear algebra objects as we typically do e.g. in the examples. How about `X` -> `X_` and `B` -> `B_` instead

Reply:
The convention used in this file is for capitals to refer to matrices and lowercase is used for vectors. There are two elimination functions which break this convention but the majority of methods follow this.

I thought the convention you refer to only applied to FEM classes.